docs/pages/9 - Contrib Modules.md

```diff
@@ -272,36 +272,240 @@ There's an [example project](https://github.com/lihaoyi/cask/tree/master/example
+`PlayTests` trait. 
 
- 
 ```scala
 // build.sc
+import mill._
+// You have to replace VERSION
+import $ivy.`com.lihaoyi::mill-contrib-playlib:VERSION`,  mill.playlib._
+
+
+object core extends PlayModule {
+    //config
+    override def scalaVersion= T{"2.12.8"}
+    override def playVersion= T{"2.7.0"}
+    override def twirlVersion= T{"1.4.0"}
+    
+    object test extends PlayTests
+}  
+``` 
+
+Using the above definition, your build will be configured to use the default Play layout:
+```text
+.
+├── build.sc
+└── core
+    ├── app
+    │   ├── controllers
+    │   └── views
+    ├── conf
+    ├── logs
+    ├── public
+    │   ├── images
+    │   ├── javascripts
+    │   └── stylesheets
+    └── test
+        └── controllers
+```
+
+The following compile dependencies will automatically be added to your build:
+```
+    ivy"com.typesafe.play::play:${playVersion()}",
+    ivy"com.typesafe.play::play-guice:${playVersion()}",
+    ivy"com.typesafe.play::play-server:${playVersion()}",
+    ivy"com.typesafe.play::play-logback:${playVersion()}"
+```
+
+Scala test will be setup as the default test framework and the following test dependencies will be 
+added (the actual version depends on the version of Play you are pulling `2.6.x` or `2.7.x`):
+```
+    ivy"org.scalatestplus.play::scalatestplus-play::4.0.1"
+```
+
+In order to have a working `start` command the following runtime dependency is also added: 
+```
+    ivy"com.typesafe.play::play-akka-http-server:${playVersion()}"
+```
+#### Using `PlayApiModule`
+
+The `PlayApiModule` trait behaves the same as the `PlayModule` trait but it won't process .scala
+.html files and you don't need to define the twirlVersion:
 
+```scala
+// build.sc
+import mill._
 // You have to replace VERSION
 import $ivy.`com.lihaoyi::mill-contrib-playlib:VERSION`,  mill.playlib._
 
-object app extends RouterModule {
-// ...
-} 
+
+object core extends PlayApiModule {
+    //config
+    override def scalaVersion= T{"2.12.8"}
+    override def playVersion= T{"2.7.0"}   
+    
+    object test extends PlayTests
+}  
 ``` 
 
-#### Configuration options
+#### Play configuration options
+
+The play modules themselves don't have specific configuration options at this point but the [router 
+module configuration options](#router-configuration-options) and the [Twirl module configuration 
+options](#twirl-configuration-options) are applicable. 
+
+#### Commands equivalence
+
+Mill commands are targets on a named build. For example if your build is called `core`:
+- compile: `core.compile`
+- run: not implemented. It can be emulated with `mill -w runBackground` but this 
+doesn't do any kind of classloading optimization and may be slower than Play's usual `run`. 
+- start: `core.start` or `core.run`
+- test: `core.test`
+- dist: not implemented, however you can use the equivalent `core.assembly` 
+command to get a runnable fat jar of the project. The packaging is slightly different though.
+
```

Comment:
I think you need to be clear there that the server is running in PROD mode and not in DEV. Obvious in hindsight maybe but not when you are trying to get everything working from scratch.

Reply:
Hopefully my new wording is clearer :)